shelley-ma/shelley-ma-test/src/Test/Cardano/Ledger/ShelleyMA/Serialisation/Roundtrip.hs

```diff
@@ -45,34 +47,36 @@ data EraIndex index where
   Mary :: EraIndex (MaryEra StandardCrypto)
   Shelley :: EraIndex (ShelleyEra StandardCrypto)
   Allegra :: EraIndex (AllegraEra StandardCrypto)
-  -- Add new Era's here, like this:
-  -- Alonzo :: EraIndex (AlonzoEra StandardCrypto)
+
+-- Add new Era's here, like this:
+-- Alonzo :: EraIndex (AlonzoEra StandardCrypto)
 
 instance Show (EraIndex e) where
   show Mary = "Mary Era"
   show Shelley = "Shelley Era"
   show Allegra = "Allegra Era"
-  -- Show Alonzo = "Alonzo Era"
+
+-- Show Alonzo = "Alonzo Era"
```

Comment:
Why not just delete this? I'm sure it will be straightforward to add when necessary :slightly_smiling_face: